plugin/move.vim

```diff
@@ -174,8 +132,39 @@ function s:MoveBlockHorizontally(distance)
     let &virtualedit = l:old_virtualedit
     let @" = l:old_default_register
 
-    " Reselect the pasted text.
-    execute "normal! g`[\<C-v>g`]"
+    return 1
+endfunction
+
+"
+" In normal mode, move the character under the cursor horizontally
+"
+function s:MoveCharHorizontally(distance)
+
```

Comment:
I'm just noticing it now but the newlines are all over the place now. I'd prefer to not have a newline after `function` and before `endfunction`.